code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -1013,16 +1013,11 @@
 			death()
 			toggle_headlamp(1)
 			return
-		if(IsUnconscious() || IsStun() || IsKnockdown() || IsParalyzed() || getOxyLoss() > maxHealth*0.5)
-			if(stat == CONSCIOUS)
-				set_stat(UNCONSCIOUS)
-				become_blind(UNCONSCIOUS_BLIND)
-				update_mobility()
+		if(HAS_TRAIT(src, TRAIT_KNOCKEDOUT) || IsStun() || IsKnockdown() || IsParalyzed())
```

Comment:
This one is fine to check rather than stat, since this is the proc that updates stat so it makes sense to use that instead here

Reply:
👍